src/module.cpp

```diff
@@ -3029,10 +3092,19 @@ int Module::CompileAndOutput(const char *srcFile, Arch arch, const char *cpu, st
         lEmitDispatchModule(dispatchModule, exportedFunctions);
 
         if (outFileName != NULL) {
-            if ((outputType == Bitcode) || (outputType == BitcodeText))
+            switch (outputType) {
+            case CPPStub:
```

Comment:
Why not writing multiple *.i files in case of multi-target? This worked fine in previous revisions?

Reply:
```suggestion
            case CPPStub:
                // No preprocessor output for dispatch module.
```